crates/libs/metadata/src/error.rs

```diff
@@ -0,0 +1,84 @@
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Default, Debug)]
+pub struct Error {
```

Comment:
`std::error::Error` is not implemented for this type, are we supposed to convert it to `std::io::Error` (and loose the span info)?

This for example prevents using the `?` operator to bubble the error up in an `anyhow::Result` chain.